src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceIT.java

```diff
@@ -231,4 +235,69 @@ public void stressTestCreateUrgentEvent()
     }
     sw.stop();
   }
+
+  private void generateDummyException(int value) throws SnowflakeSQLLoggedException {
+    if (value > 0)
+    {
+      String queryID = "01234567-1234-1234-1234-00001abcdefg";
+      String reason = "This is a test exception.";
+      String sqlState = SqlState.NO_DATA;
+      int vendorCode = 0;
+      throw new SnowflakeSQLLoggedException(queryID, reason, sqlState, vendorCode);
+    }
+  }
+  
+  /**
+   * Manual test case for checking telemetry message for SnowflakeSQLExceptions. Connect to dev or prod account and
+   * query client_telemetry table in s3testaccount for results.
+   * @throws SQLException
+   */
+  @Ignore
```

Comment:
why you decided to ignore this test?

Reply:
I don't think it makes sense to run it automatically every time because the user has to manually check if the telemetry logs are there in the client_telemetry table.

Reply:
I think it makes sense to do the client side check, i.e., the telemetry message has been sent to the oob endpoint, i.e., a valid status code has been returned from it. WDYT?

Reply:
Sounds good, I added a check for connecting to oob endpoint!